test/unit/ensure_not_used_by_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'test_helper'
+
+class EnsureNotUsedByTest < ActiveSupport::TestCase
+  def setup
+    @org1 = FactoryGirl.create(:organization)
+    @org2 = FactoryGirl.create(:organization)
+    @user = User.create :auth_source => auth_sources(:one), :login => "foo", :mail  => "foo@bar.com"
```

Comment:
Use a factory for this, e.g. `FactoryGirl.create(:user, :with_mail, :organizations => [@org1])`